Makefile

```diff
@@ -1,46 +1,23 @@
-# .RECIPEPREFIX +=
 CC = gcc
-SRCS = src/ftsh.c src/ftsh_functions.c
+
+# Add any files that need to be compiled into the executable below in SRCS
+SRCS = src/ftsh.c src/ftsh_functions.c src/search.c src/match.c
 OBJS = $(SRCS:.c=.o)
-FTSH_BIN = ftsh
+BIN = ftsh
+CFLAGS = -g -O2 -Wall -std=c99 -I ./include/ -I ./src/ -fPIC `pkg-config --cflags --libs glib-2.0`
```

Comment:
What do we need these `pkg-config` lines for? What functionality do you need from `glib-2.0`?

Reply:
Removed!